compiler.config.json

```diff
@@ -0,0 +1,524 @@
+            "Agreements/Internal/TCWEL_TCCOW.txt",
+            "Agreements/Internal/Trent_East.txt",
+            "Agreements/Internal/Trent_North.txt",
+            "Agreements/Internal/Trent_S29.txt",
+            "Agreements/Internal/Trent_Stafa.txt",
+            "Agreements/Internal/Tyne_Mon.txt",
+            "Agreements/Internal/Wal_5.txt",
+            "Agreements/Internal/Wal_IoM.txt",
+            "Agreements/Internal/Wal_North.txt",
+            "Agreements/Internal/Wal_Stafa.txt"
+        ],
+        "sector": [
+            "Airports/EGAA/Ownership.txt",
+            "Airports/EGAC/Ownership.txt",
+            "Airports/EGAE/Ownership.txt",
+            "Airports/EGBB/Ownership.txt",
+            "Airports/EGBJ/Ownership.txt",
+            "Airports/EGCC/Ownership.txt",
+            "Airports/EGCN/Ownership.txt",
+            "Airports/EGFF/Ownership.txt",
+            "Airports/EGGD/Ownership.txt",
+            "Airports/EGGP/Ownership.txt",
+            "Airports/EGGW/Ownership.txt",
+            "Airports/EGHC/Ownership.txt",
+            "Airports/EGHE/Ownership.txt",
+            "Airports/EGHH/Ownership.txt",
+            "Airports/EGHI/Ownership.txt",
+            "Airports/EGHQ/Ownership.txt",
+            "Airports/EGJA/Ownership.txt",
+            "Airports/EGJB/Ownership.txt",
+            "Airports/EGJJ/Ownership.txt",
+            "Airports/EGKA/Ownership.txt",
+            "Airports/EGKB/Ownership.txt",
+            "Airports/EGKK/Ownership.txt",
+            "Airports/EGLC/Ownership.txt",
+            "Airports/EGLF/Ownership.txt",
+            "Airports/EGLL/Ownership.txt",
+            "Airports/EGMC/Ownership.txt",
+            "Airports/EGMD/Ownership.txt",
+            "Airports/EGNC/Ownership.txt",
+            "Airports/EGNH/Ownership.txt",
+            "Airports/EGNJ/Ownership.txt",
+            "Airports/EGNM/Ownership.txt",
+            "Airports/EGNS/Ownership.txt",
+            "Airports/EGNT/Ownership.txt",
+            "Airports/EGNV/Ownership.txt",
+            "Airports/EGNX/Ownership.txt",
+            "Airports/EGPA/Ownership.txt",
+            "Airports/EGPB/Ownership.txt",
+            "Airports/EGPC/Ownership.txt",
+            "Airports/EGPD/Ownership.txt",
+            "Airports/EGPE/Ownership.txt",
+            "Airports/EGPF/Ownership.txt",
+            "Airports/EGPH/Ownership.txt",
+            "Airports/EGPK/Ownership.txt",
+            "Airports/EGPM/Ownership.txt",
+            "Airports/EGPN/Ownership.txt",
+            "Airports/EGPO/Ownership.txt",
+            "Airports/EGSC/Ownership.txt",
+            "Airports/EGSH/Ownership.txt",
+            "Airports/EGSS/Ownership.txt",
+            "Airports/EGSY/Ownership.txt",
+            "Airports/EGTC/Ownership.txt",
+            "Airports/EGTE/Ownership.txt",
+            "Airports/EGTK/Ownership.txt",
+            "Airports/EGWU/Ownership.txt",
+            "Ownership/AC - Central/12.txt",
```

Comment:
Does this dictate the order the files are put into the .ese? If so, order matters for ownership. The current file puts LTC -> MPC -> STC -> AC -> ScAC -> Non-UK - smaller sectors generally need to be above bigger sectors

Reply:
Yes it does - no problem, I can fix that!